src/System.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -62,8 +62,7 @@ public static void LocalDBNotSupportedOnUapTest()
             builder.IntegratedSecurity = true;
             builder.ConnectTimeout = 2;
 
-            string errorMsg = SystemDataResourceManager.Instance.LocalDBNotSupported;
```

Comment:
.. .based on the assumption the others weren't hitting it (as I recall). But now they are: https://mc.dot.net/#/product/netcore/master/source/official~2Fcorefx~2Fmaster~2F/type/test~2Ffunctional~2Filc~2F/build/20170724.02/workItem/System.Data.SqlClient.Tests/analysis/xunit/System.Data.SqlClient.Tests.SqlErrorCollectionTest~2FSyncRoot_Success